src/iperf_api.c

```diff
@@ -1379,6 +1382,51 @@ iperf_parse_arguments(struct iperf_test *test, int argc, char **argv)
         warning("Debug output (-d) may interfere with JSON output (-J)");
     }
 
+    if (test->settings->lso_udp) {
```

Comment:
not all linuxes support GSO_UDP socket option. We should somehow query that before use.

Reply:
I've made a check for supporting (iperf_api.c - line 98). But it works only at compile time. In runtime, using of UDP LSO will cause the error in "sendmsg" call - EINVAL.

Reply:
think that makes sense. EINVAL is enouth